docs/design/2020-06-24-placement-rules-in-sql.md

```diff
@@ -20,196 +45,193 @@ The scenarios of defining placement rules in SQL include:
 - Increase the replica count of more important data
 - Separate irrelevant data into different stores to improve availability
 
-## Define placement rules
+These scenarios usually fit into one of the following categories:
```

Comment:
> - Place latest data to SSD and history data to HDD
> - Place the leader of hot data to a high-performance TiKV instance

These scenarios is not covered in this document, since it seem to only cover whole database/tables/partitions/indexes and not ranges/list of data within. Also the first would probably need some kind of dynamic way of transition data from 'latest' to 'old'.
If the idea is that partitioning management will cover this, then I think it should be explicit (and give some time to improve).

Reply:
I have added a specific section for examples. Can you comment if you'd like to see any specific examples added?